tests/config/test_data.config

```diff
@@ -40,15 +40,19 @@ params {
                 md = "${params.test_data_base}/main/.test_data/md.nii.gz"
                 priors = "${params.test_data_base}/main/.test_data/priors"
             }
-        }
-        "reconst" {
             "noddi"{
                 dwi = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize.nii.gz"
                 mask = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize_mask.nii.gz"
                 bval = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize_bval.bval"
                 bvec = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize_bvec.bvec"
                 kernels = "${params.test_data_base}/main/.test_data/kernels"
             }
+            "dtimetrics"{
+                dwi = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize.nii.gz"
+                b0mask = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize_mask.nii.gz"
+                bval = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize_bval.bval"
+                bvec = "${params.test_data_base}/main/.test_data/preproc/dwi_normalize_bvec.bvec"
```

Comment:
Shouldn't this points towards 'reconst' rather than 'preproc' ?

Reply:
No, I'm just using the data in the preproc module because I can't implement new data.